Product/Production/Services/DocumentQueryCore/pom.xml

```diff
@@ -117,5 +117,57 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.connectopensource</groupId>
+            <artifactId>AuditRepositoryCore</artifactId>
+            <version>${project.parent.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+
     </dependencies>
```

Comment:
I'll admit this one is a nitpick, but, if you're going to have an extra blank line, can you put it after </dependencies> instead of before?